python_modules/dagster/dagster_tests/components_tests/integration_tests/test_definitions_autoload.py

```diff
@@ -151,7 +156,12 @@ def test_ignored_empty_dir():
     with create_project_from_components(path_str) as (project_root, project_name):
         module = importlib.import_module(f"{project_name}.defs.{src_path.stem}")
         context = ComponentLoadContext.for_module(module, project_root)
-        root = DefsFolderComponent.get(context)
-        for comp in root.iterate_components():
+        root = check.inst(get_component(context), CompositeYamlComponent)
+        defs_root = check.inst(root.components[0], DefsFolderComponent)
```

Comment:
a little help that gets a singular component would help the test cleanliness problem